lib/ftoggle.js

```diff
@@ -22,60 +23,60 @@ class Ftoggle {
   }
 
   doesFeatureExist(feature) {
-    return _.has(this.toggleConfig, this.makeFeaturePath(feature));
+    return _.has(this.featureConfig, this.makeFeaturePath(feature));
   }
 
-  getFeatures() {
-    return this.config;
+  getToggles() {
+    return this.toggles;
   }
 
-  featureVal(key) {
-    return this.featureVals[key] || null;
+  getSetting(key) {
+    return this.settings[key] || null;
   }
 
-  getFeatureVals() {
-    return this.featureVals;
+  getSettings() {
+    return this.settings;
   }
 
-  getConfigForFeature(feature) {
-    return _.get(this.toggleConfig, `${feature}.conf`);
+  getSettingsForFeature(feature) {
```

Comment:
is there ever a chance of feature being `null` or `undefined`?  what would happen for
```
`${feature}.settings`
```

would it just be ... ` null.settings`

Reply:
I think it might actually blow up, but we pretty much only call this internally.